api/handlers_instances.go

```diff
@@ -498,7 +498,7 @@ func (s *server) InstanceSSMAssociationHandler(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	handleResponseOk(w, struct{ AssociationId string }{AssociationId: *out.AssociationDescription.AssociationId})
+	handleResponseOk(w, aws.StringValue(out.AssociationDescription.AssociationId))
```

Comment:
I think your function should only return the AssociationId in `out` so we don't need to parse this in the handler.
That's what we've done for all the other functions.